cmake/gadgetron-config.cmake.in

```diff
@@ -1,24 +1,25 @@
 @PACKAGE_INIT@
-
-
-find_package(PythonLibs 3  REQUIRED)
-find_package(Boost 1.65.0 COMPONENTS system program_options filesystem timer REQUIRED )
-if (Boost_VERSION_STRING VERSION_LESS 1.67.0)
-    find_package(Boost 1.65.0 COMPONENTS python3 REQUIRED)
-    set(Boost_PYTHON3_TARGET Boost::python3)
+   
+if (BUILD_PYTHON_SUPPORT)
+    find_package(PythonLibs 3  REQUIRED)
+    if (Boost_VERSION_STRING VERSION_LESS 1.67.0)
+        find_package(Boost 1.65.0 COMPONENTS python3 REQUIRED)
+        set(Boost_PYTHON3_TARGET Boost::python3)
     else()
-    string(REGEX MATCH "^3\\.([0-9]+)\\.[0-9]+" PYTHON_MINOR_VERSION ${PYTHONLIBS_VERSION_STRING} )
-    set(PYTHON_MINOR_VERSION ${CMAKE_MATCH_1})
-    find_package(Boost 1.65.0 COMPONENTS "python3${PYTHON_MINOR_VERSION}" REQUIRED)
-    set(Boost_PYTHON3_FOUND TRUE)
-    set(Boost_PYTHON3_TARGET Boost::python3${PYTHON_MINOR_VERSION})
+        string(REGEX MATCH "^3\\.([0-9]+)\\.[0-9]+" PYTHON_MINOR_VERSION ${PYTHONLIBS_VERSION_STRING} )
+        set(PYTHON_MINOR_VERSION ${CMAKE_MATCH_1})
+        find_package(Boost 1.65.0 COMPONENTS "python3${PYTHON_MINOR_VERSION}" REQUIRED)
+        set(Boost_PYTHON3_FOUND TRUE)
+        set(Boost_PYTHON3_TARGET Boost::python3${PYTHON_MINOR_VERSION})
+    endif()
+    set(GADGETRON_INSTALL_PYTHON_MODULE_PATH "@GADGETRON_INSTALL_PYTHON_MODULE_PATH@" )
 endif()
 
+find_package(Boost 1.65.0 COMPONENTS system program_options filesystem timer REQUIRED )
 
 add_definitions(-DARMA_DONT_USE_WRAPPER -DARMA_USE_CXX11 -DARMA_64BIT_WORD)
 set(GADGETRON_INSTALL_CONFIG_PATH "@GADGETRON_INSTALL_CONFIG_PATH@" )
 set(GADGETRON_INSTALL_SCHEMA_PATH "@GADGETRON_INSTALL_SCHEMA_PATH@" )
-set(GADGETRON_INSTALL_PYTHON_MODULE_PATH "@GADGETRON_INSTALL_PYTHON_MODULE_PATH@" )
```

Comment:
This is the line I'm concerned about removing in the no-python case - I agree it should probably go, but I'm not sure its a safe change without increasing the scope of this PR.